packages/amplify-codegen/src/utils/getOutputFileName.js

```diff
@@ -1,18 +1,27 @@
 const path = require('path');
 
-function getOutputFileName(inputFileName = 'API', target) {
+function getOutputFileName(inputFileName, target) {
   const extensionMap = {
     swift: 'swift',
     typescript: 'ts',
     flow: 'js',
+    angular: 'service.ts',
```

Comment:
Is this the extension or the filename? the values for swift/typescript/flow etc looks like just extensions, unlike that for angular

Reply:
Yes. Angular adds `service.ts` to Services